### PR TITLE
W-11540242-allowlisting-of-trusted-domains-fa

### PR DESCRIPTION
**Pull request by failup**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/allowlisting-trusted-domains.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Every HTTP request you make to an external resource goes through the API Console Proxy. For security reasons, you need to register the domain as a trusted domain in Anypoint Platform.
+
+To add a new domain:
+
+1. Go to Access Management.
+2. Click *Trusted domains*.
+3. Click *Add domain*. +
+The domain must not match with any existing trusted domain, and must not be any domain or subdomain of Mulesoft or Anypoint Platform.
+4. In the dialog box that appears, type the domain name and click *Add domain*. +
+_Result_: The domain has been added to your list of trusted domains. You are redirected to the trusted domains page, where you can edit or delete your domains. If you register a domain on a parent organization, it will also be allowlisted on all your business groups.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/allowlisting-trusted-domains.adoc]:
```suggestion
include::partial$include-nav-steps-org-admin.adoc[]

. Click *Trusted domains*.
. Click *Add domain*. +
The domain must not match with any existing trusted domain, and must not be any domain or subdomain of MuleSoft or Anypoint Platform. 
+
. In the dialog box that appears, type the domain name and click *Add domain*. +
_Result_: The domain has been added to your list of trusted domains. You are redirected to the trusted domains page, where you can edit or delete your domains. If you register a domain on a parent organization, it is also allowlisted on all your business groups.
```

File: modules/ROOT/pages/allowlisting-trusted-domains.adoc [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+== Automatic Registration of Domains
+When publishing an API specification to Exchange, or sending a request trough the proxy, the base URI of the implementation will automatically be added to the list of trusted domains. Administrators will receive an email notifying you of this event. The email will redirect you to the *Trusted domains* page in Access Management, where you can add or delete new domains as explained before.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/allowlisting-trusted-domains.adoc]:
```suggestion

When you publish an API specification to Exchange or send a request through the proxy, the base URI of the implementation is automatically added to the list of trusted domains. Organization administrators receive an email notification stating a trusted domain has been added, and the email redirects them to the *Trusted domains* page in Access Management, where they can add or delete new domains.
```

File: modules/ROOT/pages/allowlisting-trusted-domains.adoc [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+NOTE: The allowlisting of trusted domains will be rolled in two phases. During the first phase, non-authenticated request will be automatically registered. During the second phase, non-authenticated requests from public portals or via link share won't be automatically added to the list of trusted domains and will fail if the domain is not manually added to the list.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/allowlisting-trusted-domains.adoc]: We can't promise any kind of future functionality for the platform;  we will have to rethink how this is written, as it cannot be published in this state.

Comment (failup) [reply modules/ROOT/pages/allowlisting-trusted-domains.adoc]: Would it be okay to leave the note stating how it works now? e.g: Note: non-authenticad requests will be automatically registered

Comment (sarathecoubian) [reply modules/ROOT/pages/allowlisting-trusted-domains.adoc]: Yes, as long as it's changed to present tense :)

File: modules/ROOT/pages/allowlisting-trusted-domains.adoc [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+The domain must not match with any existing trusted domain, and must not be any domain or subdomain of MuleSoft or Anypoint Platform. 
++
+. In the dialog box that appears, type the domain name and click *Add domain*. +
+_Result_: The domain has been added to your list of trusted domains. You are redirected to the trusted domains page, where you can edit or delete your domains. If you register a domain on a parent organization, it is also allowlisted on all your business groups.
+
+== Automatic Registration of Domains
+
+When you publish an API specification to Exchange or send a request through the proxy, the base URI of the implementation is automatically added to the list of trusted domains. Organization administrators receive an email notification stating a trusted domain has been added, and the email redirects them to the *Trusted domains* page in Access Management, where they can add or delete new domains.
+
+NOTE: Non-authenticated requests requests from public portals or via link share are automatically registered.
```
Comment (dfmacias) [inline modules/ROOT/pages/allowlisting-trusted-domains.adoc]:
```suggestion
NOTE: Non-authenticated requests from public portals in Exchange or via shareable links in Mocking Service are automatically registered. 
```

**Review by dfmacias — CHANGES_REQUESTED** [review]

**Review by dfmacias — DISMISSED** [review]

**Review by Crispy-Salesforce — DISMISSED** [review]

**Review by JennyHajee — APPROVED** [review]

**Pull request merged**